### ISSUE TITLE
tiny part of the track appears behind progress

### ISSUE THREAD

**Issue by murad1981** [yannickl/YLProgressBar#63]
when the progress view type is rounded with gradient colors, there is a tiny part of the background track view appears on top of the progress view 

![track](https://user-images.githubusercontent.com/1199045/47613120-bd904400-da99-11e8-9b40-1870c5239e0e.png)

**Comment by azablotskiy** [comment]
Have same issue :(

**Comment by rahulvyas** [comment]
+1

**Comment by rahulvyas** [comment]
Any one knows any fix ?

**Comment by rahulvyas** [comment]
Found the fix- set progressView.hideGloss = YES or true
